public/app/features/live/centrifuge/service.ts

```diff
@@ -268,6 +275,35 @@ export class CentrifugeService implements CentrifugeSrv {
     });
   }
 
+  /**
+   * Execute a query over the live websocket and potentiall subscribe to a live channel.
+   *
+   * Since the initial request and subscription are on the same socket, this will support HA setups
+   */
+  getQueryData(options: LiveQueryDataOptions): Observable<DataQueryResponse> {
+    return new Observable((subscriber) => {
+      this.centrifuge
+        .rpc(options.body)
```

Comment:
@FZambia is this scoped at all?  Currently just passing the identical body here -- but I suspect we may want to support multiple methods

Reply:
Yeah, there is a `namedRPC` method, which should become just an `rpc` in the next major release.